src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php

```diff
@@ -54,6 +54,14 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('web_profiler.debug_toolbar.mode', $config['toolbar'] ? WebDebugToolbarListener::ENABLED : WebDebugToolbarListener::DISABLED);
         }
 
+        if ($config['intercept_redirects']) {
```

Comment:
What about moving the deprecation to the `Configuration` class? That's how we do it when possible.

Reply:
Done.